Hangman/Hangman/main.swift

```diff
@@ -2,11 +2,167 @@
+let hangManDrawing = ["""
+-----|
+|
+|
+|
+|
+|
+--------
+""",
+                      """
+-----|
+|    O
+|
+|
+|
+|
+--------
+""",
+                      """
+-----|
+|    O
+|    |
+|
+|
+|
+--------
+""",
+                      """
+-----|
+|    O
+|    |\\
+|
+|
+|
+---------
+""",
+                      """
+-----|
+|    O
+|   /|\\
+|
+|
+|
+---------
+""",
+                      """
+-----|
+|    O
+|   /|\\
+|     \\
+|
+|
+---------
+""",
+                      """
+-----|
+|    O
+|   /|\\
+|   / \\
+|
+|
+---------
+"""
+]
 
-print("Hello, World!")
+//var gameState = true
+//var gameRestart = false
+var repeatOnlyForAttempts = true
 
+
+func Hangman() {
+
+    var numOfLosses = 0
+    var numOfAttemptsRemaining = 6
+    let randomAiWord = allTheWords.randomElement()
+    var validAiWordAsString = ""
+    
+    if let validRandomAiWord = randomAiWord {
+        validAiWordAsString = validRandomAiWord
+    } else {
+        print("There is no random word to choose from")
+    }
+    
+    print("You have a total of \(numOfAttemptsRemaining) attempts to guess a random word that the computer chooses or else you'll lose the game")
```

Comment:
This would have been a good place to display their previous guessed letters as well